tests/admin_changelist/tests.py

```diff
@@ -1686,3 +1686,72 @@ def test_save_without_changes_warns_on_pending_action(self):
             )
         finally:
             alert.dismiss()
+
+
+@modify_settings(MIDDLEWARE={'remove': 'django.contrib.admin.tests.CSPMiddleware'})
+@override_settings(ROOT_URLCONF='admin_changelist.urls')
+class AdminFiltersSeleniumTests(AdminSeleniumTestCase):
+
+    available_apps = ['admin_changelist'] + AdminSeleniumTestCase.available_apps
+
+    def setUp(self):
+        User.objects.create_superuser(username='super', password='secret', email=None)
+
+    def test_collapse_filters(self):
+        from selenium.webdriver.common.by import By
+
+        from django.contrib.auth.models import Group as AdminGroup
```

Comment:
I imported here because the rest of the tests use `Group`, but it's the "Band Group" not the auth model

Reply:
You can move it to the top, because we have an alias.